src/app/game-state/battle.service.ts

```diff
@@ -144,7 +144,11 @@ export class BattleService {
       for (let i = 0; i < enemyStack.quantity; i++){
         if (Math.random() < enemyStack.enemy.accuracy){
           let damage = enemyStack.enemy.attack;
-          damage = damage / (1 + this.characterService.characterState.defense * 0.1);
+          let combatDefense = this.characterService.characterState.defense;
```

Comment:
The old equation would make defense make Death's effective health in the 18th order of magnitude, far beyond what players can make weapons do in a real-time century.